package/htmd-deps/write_meta_yaml.py

```diff
@@ -25,18 +25,26 @@
 deps = []
 with open(os.path.join(workdir, 'DEPENDENCIES'), 'r') as f:
     for line in f:
-        deps.append(line.strip().lower())
+        # Remove comments and split
+        packagematch = line.split('#')[0].split()
+        # Append dict of the conda package match specification
+        if packagematch:
+            deps.append(dict(zip(['name', 'version', 'build_string'], packagematch)))
```

Comment:
Doesn´t this assume that the split will return 3 values? I think zip will fail on lists of different lengths no?

Reply:
It surprisingly works, I tested it (and it passes Travis)! But if you think it's unsafe, please let me know how to do it.

Reply:
You are right. It apparently works
```
>>> list(zip(['name', 'value', 'index'], ['free', 'test']))
[('name', 'free'), ('value', 'test')]
```
Personally I don't like this behaviour of python but that's irrelevant to the PR.